Remotion/Web/Core/Doc/include/Configuration/ResourceRoot.xml

```diff
@@ -15,34 +15,33 @@
  ! You should have received a copy of the GNU Lesser General Public License
  ! along with re-motion; if not, see http://www.gnu.org/licenses.
 -->
-<ResourceConfiguration>
+<ResourceRoot>
   <Class>
     <remarks> 
       See the <see cref="WebConfiguration"/> for details on registering the <c>remotion.web</c> configuration section.
     </remarks>
     <seealso cref="WebConfiguration"/>
     <seealso cref="Remotion.Web.ResourceUrlResolver"/>
     <example>
-      Setting the <c>resources</c> entry's values in the <c>remotion.web</c> configuration section.
+      Configuring the resource root is done by adding a new implementation of
+      <c>ResourceRoot</c>, overriding it's <c>Value</c> field and putting
+      it into the IOC container.
+
       <code>
 <![CDATA[
-<remotion.web xmlns="http://www.re-motion.org/web/configuration"> 
-  <resources 
-      root="res" 
-      rootPath="AbsoluteWithApplicationRoot" 
-  />
-  <!-- Other configuration section entries. -->
-</remotion.web>
-]]></code>
+  [ImplementationFor(typeof(ResourceRoot), Lifetime = LifetimeKind.Singleton)]
+  public class MyOwnResourceRoot : ResourceRoot
+  {
+
+    public override string Value => "myResourceRoot";
+  }
+]]>   </code>
     </example>
   </Class>
-  <Root>
+  <RootValue>
```

Comment:
This should be the propertyname, exactly. given how little text there is, inlining it might be the better choice? The separate doc files where intended to make it easier to maintain the docs and declutter the classes. These days, they are more trouble than they're worth for anything that's just a couple of lines of text.

Reply:
We just need to make sure that `<Value>` does not conflict with the `<value>` XML tag that exists in XML docs. Don't know if this is a problem here.

Reply:
Don't this there's a conflict. It's `ResourceRoot/Value`, that's a unique xpath.